components/KiboOrderItemDetails.vue

```diff
@@ -4,81 +4,116 @@
-          <div class="order-shipment-items"></div>
-        </div>
-      </div>
-      <div class="order-payment">
-        <div class="order-payment-header"></div>
-        <div class="order-payment-method">
-          <div class="order-payment-title order-item-box"></div>
-          <div class="order-payment-info order-item-box"></div>
-        </div>
-      </div>
-      <div class="order-billing-address order-item-box"></div>
-      <hr class="details-hr-normal" />
-      <div class="order-shipping-address order-item-box"></div>
-      <hr class="details-hr-normal" />
-      <div class="order-summary order-item-box">
-        <div class="order-summary-header"></div>
+      <div v-if="order.items" class="order-shipment">
+        <KiboOrderLineItems :order="order">
+          <template #ship-title>
+            <h3 class="sf-heading__title h3">{{ $t("Shipment Details") }}</h3>
+          </template>
+          <template #ship-sub-section>
+            <div class="order-shipment__section">
+              <div class="order-shipment__info">{{ order.status }}</div>
+              <div class="order-shipment__status">{{ orderExpectedDeliveryDate }}</div>
+            </div>
+          </template>
+          <template #pickup-title>
+            <h3 class="sf-heading__title h3">{{ $t("Pickup") }}</h3>
+          </template>
+          <template #pickup-sub-section>
+            <div class="order-shipment__section">
+              <div class="order-shipment__info">{{ $t("Pickup In Store") }}</div>
+              <div class="order-shipment__status">
+                {{ $t("Est. Pickup:") }} {{ orderExpectedDeliveryDate }}
+              </div>
+            </div>
+          </template>
+        </KiboOrderLineItems>
       </div>
     </div>
   </div>
 </template>
 <script lang="ts">
-import { defineComponent } from "@vue/composition-api"
+import { defineComponent, PropType } from "@vue/composition-api"
+import { SfProperty } from "@storefront-ui/vue"
+import { orderGetters } from "@/lib/getters"
+import { Order } from "@/server/types/GraphQL"
 
 export default defineComponent({
   name: "KiboOrderItemDetails",
+  components: {
+    SfProperty,
+  },
   props: {
     order: {
-      type: Object,
+      type: Object as PropType<Order>,
       default: () => ({}),
     },
   },
-  setup() {},
+  setup(props) {
+    const orderNumber = computed(() => {
+      return orderGetters.getOrderNumber(props.order)
+    })
+    const orderId = computed(() => {
+      return orderGetters.getId(props.order)
+    })
+    const orderSubmittedDate = computed(() => {
+      return orderGetters.getSubmittedDate(props.order, true)
+    })
+    const orderTotal = computed(() => {
+      return orderGetters.getOrderTotal(props.order)
+    })
+    const orderStatus = computed(() => {
+      return orderGetters.getOrderStatus(props.order)
+    })
+    const orderExpectedDeliveryDate = computed(() => {
+      return orderGetters.getExpectedDeliveryDate(props.order)
+    })
```

Comment:
Can we make it one-liner, it will be more compact and readable?
ex.  const orderExpectedDeliveryDate = computed(() => orderGetters.getExpectedDeliveryDate(props.order)
    )

Reply:
Done